tests/clients/test_environment.py

```diff
@@ -70,3 +339,90 @@ def test_environment_client_list(mocker):
     EnvironmentClient._get.assert_called_once_with(
         "/project/{}/environment".format(PROJECT_ID), schema_mock.return_value
     )
+
+
+def test_environment_client_get(mocker):
+    mocker.patch.object(EnvironmentClient, "_get", return_value=ENVIRONMENT)
+    schema_mock = mocker.patch("faculty.clients.environment.EnvironmentSchema")
+
+    client = EnvironmentClient(mocker.Mock())
+    assert client.get(PROJECT_ID, ENVIRONMENT_ID) == ENVIRONMENT
+
+    schema_mock.assert_called_once_with()
+    EnvironmentClient._get.assert_called_once_with(
+        "/project/{}/environment/{}".format(PROJECT_ID, ENVIRONMENT_ID),
+        schema_mock.return_value,
+    )
+
+
+def test_environment_client_update(mocker):
+    mocker.patch.object(EnvironmentClient, "_put_raw", return_value=None)
```

Comment:
```suggestion
    mocker.patch.object(EnvironmentClient, "_put_raw")
```
Don't need this as the return value is not used.